tsconfig.json

```diff
@@ -7,11 +7,12 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "outDir": "public/build"
```

Comment:
This line is what configures the transpiled typescript files' target directory. In this case all transpiled files will be output into public/build.